From 57d7d4c9c8d7b44f402969be058ff80e7622c660 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Thu, 22 Nov 2018 22:28:07 +0530
Subject: [PATCH 7/9] Always generate a pkg-config file and don't require
 latest bash

Shell script uses `declare -n` but our bash is too old to support
that. Luckily, that codepath is only used on Cygwin, so we can make it
a no-op.
---
 build/make/msvs_common.sh | 2 +-
 libs.mk                   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/make/msvs_common.sh b/build/make/msvs_common.sh
index 88f1cf9..84c1ca5 100644
--- a/build/make/msvs_common.sh
+++ b/build/make/msvs_common.sh
@@ -41,7 +41,7 @@ fix_path() {
 # Corrects the paths in file_list in one pass for efficiency.
 # $1 is the name of the array to be modified.
 fix_file_list() {
-    declare -n array_ref=$1
+    declare array_ref=$1
     files=$(fix_path "${array_ref[@]}")
     local IFS=$'\n'
     array_ref=($files)
diff --git a/libs.mk b/libs.mk
index b983aed..4e8fc28 100644
--- a/libs.mk
+++ b/libs.mk
@@ -318,6 +318,7 @@ INSTALL-LIBS-$(CONFIG_SHARED) += $(LIBVPX_SO_SYMLINKS)
 INSTALL-LIBS-$(CONFIG_SHARED) += $(LIBSUBDIR)/$(LIBVPX_SO)
 INSTALL-LIBS-$(CONFIG_SHARED) += $(if $(LIBVPX_SO_IMPLIB),$(LIBSUBDIR)/$(LIBVPX_SO_IMPLIB))
 
+endif
 
 LIBS-yes += vpx.pc
 vpx.pc: config.mk libs.mk
@@ -343,7 +344,6 @@ endif
 INSTALL-LIBS-yes += $(LIBSUBDIR)/pkgconfig/vpx.pc
 INSTALL_MAPS += $(LIBSUBDIR)/pkgconfig/%.pc %.pc
 CLEAN-OBJS += vpx.pc
-endif
 
 #
 # Rule to make assembler configuration file from C configuration file
-- 
2.20.1.windows.1

